server/server.go

```diff
@@ -156,7 +156,7 @@ func SetupServerRoutes(appConfig *config.AppConfig, services *config.AppServices
 		sharedConfigRoutes,
 	}
 
-	if appConfig.ExperimentalFeatures.PikeEnabled {
+	if appConfig.ExperimentalFeatures.PikeContactsEnabled || appConfig.ExperimentalFeatures.PikePaymentEnabled {
```

Comment:
I'm not sure about that. 
I thought, "contacts" itself depends of `PikeContactsEnabled` only 🤔

Reply:
Done